Missionframework/modules/16_garrison/fnc/fn_garrison_dialogRemove.sqf

```diff
@@ -0,0 +1,101 @@
+#include "..\ui\defines.hpp"
+/*
+    KPLIB_fnc_garrison_dialogRemove
+
+    File: fn_garrison_dialogRemove.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2019-03-31
+    Last Update: 2019-03-31
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+    Public: No
+
+    Description:
+        Spawns infantry/vehicle and removes it from garrison array.
+
+    Parameter(s):
+        _button - Pressed button control [CONTROL, defaults to controlNull]
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+params [
+    ["_button", controlNull, [controlNull]]
+];
+
+if (isNull _button) exitWith {false};
+
+// Dialog controls
+private _dialog = findDisplay KPLIB_IDC_GARRISON_DIALOG;
+private _ctrlLbSectors = _dialog displayCtrl KPLIB_IDC_GARRISON_GARRISONLIST;
+private _ctrlInfButton = _dialog displayCtrl KPLIB_IDC_GARRISON_INFANTRYBUTTON;
+private _ctrlLightButton = _dialog displayCtrl KPLIB_IDC_GARRISON_LIGHTBUTTON;
+private _ctrlHeavyButton = _dialog displayCtrl KPLIB_IDC_GARRISON_HEAVYBUTTON;
+
+// Initialize needed local variables
+private _sector = _ctrlLbSectors lbData (lbCurSel _ctrlLbSectors);
+private _sectorPos = [_sector] call KPLIB_fnc_common_getPos;
+private _garrison = [_sector] call KPLIB_fnc_garrison_getGarrison;
+private _side = KPLIB_preset_sideF;
+
+// Handle Infantry
+if (_button == _ctrlInfButton) exitWith {
```

Comment:
`isEqualTo` ?

Reply:
`isEqualTo` doesn't take controls for comparison. So it has to be `==`.